modules/restore-replace-stopped-etcd-member.adoc

```diff
@@ -350,4 +352,40 @@ If the output from the previous command only lists two pods, you can manually fo
 $ oc patch etcd cluster -p='{"spec": {"forceRedeploymentReason": "recovery-'"$( date --rfc-3339=ns )"'"}}' --type=merge <1>
 ----
 <1> The `forceRedeploymentReason` value must be unique, which is why a timestamp is appended.
+
+. Verify that there are exactly three etcd members.
+
+.. Connect to the running etcd container, passing in the name of a pod that was not on the affected node:
```

Comment:
s/passing in/specifying

Reply:
The same language is used earlier in the procedure, but I'll consider whether to update them all in a future update. Thanks!